bin/run-addon

```diff
@@ -115,7 +115,7 @@ if [[ -z "$no_auto" ]] ; then
 fi
 
 cd build/addon
-jpm run --binary $binary --profile ../../Profile --no-copy \
+jpm run --binary $binary --profile ../../dist/Profile --no-copy \
```

Comment:
Since we're putting the build files in `build/` now (which arguably is an unnecessary rename), I was presuming that we'd move profiles around to eliminate the dist directory.